### ISSUE TITLE
Type 'unique symbol' is not assignable to type 'number'

### ISSUE THREAD

**Issue by scanf** [syumai/dejs#18]
Hei,

I was checking out Dinatra and tried using dejs. It was working for me but after upgrading to most recent deno I see the errors below, similiar to the ones from the latest [CI failure](https://travis-ci.org/syumai/dejs/builds/540882570). Do you have a fix for this?

Cool project, thanks!

```
deno run --allow-read --allow-net src/main.ts
[0/1] Compiling file:///Users/scanf/src/github.com/scanf/uuid.alemayhu.com/src/main.tshttps://raw.githubuserconte
nt.com/syumai/dejs/master/dejs.ts:99:9 - error TS2469: The '<' operator cannot be applied to type 'symbol'.

99     if (byte < 0) {
           ~~~~
https://raw.githubusercontent.com/syumai/dejs/master/dejs.ts:103:14 - error TS2345: Argument of type 'number | un
ique symbol' is not assignable to parameter of type 'number'.
  Type 'unique symbol' is not assignable to type 'number'.

103     buf.push(byte);
                 ~~~~

make: *** [hello] Error 1
Alexanders-MacBook-Pro:uuid.alemayhu.com scanf$ make
deno run --allow-read --allow-net src/main.ts
[0/1] Compiling file:///Users/scanf/src/github.com/scanf/uuid.alemayhu.com/src/main.tshttps://raw.githubuserconte
nt.com/syumai/dejs/master/dejs.ts:99:9 - error TS2469: The '<' operator cannot be applied to type 'symbol'.

99     if (byte < 0) {
           ~~~~
https://raw.githubusercontent.com/syumai/dejs/master/dejs.ts:103:14 - error TS2345: Argument of type 'number | un
ique symbol' is not assignable to parameter of type 'number'.
  Type 'unique symbol' is not assignable to type 'number'.

103     buf.push(byte);
```

**Comment by syumai** [comment]
Thank you for reporting this!
I'll take a look at this.

**Comment by syumai** [comment]
This PR causes the problem. I'll fix this
https://github.com/denoland/deno_std/pull/472

**Comment by syumai** [comment]
@scanf Resolved in #19, thanks!

**Comment by scanf** [comment]
Can confirm it's working.

Thank you.
